pkg/cnab/config-adapter/adapter_test.go

```diff
@@ -636,18 +637,43 @@ func TestManifestConverter_generateDependenciesv2(t *testing.T) {
 	t.Parallel()
 
 	testcases := []struct {
-		name    string
-		wantDep depsv2ext.Dependency
+		name     string
+		wantDep  depsv2ext.Dependency
+		wantDefs definition.Definitions
 	}{
-		{"all fields", depsv2ext.Dependency{
+		{name: "all fields", wantDep: depsv2ext.Dependency{
```

Comment:
Doesn't look like this has gone through go fmt.

Reply:
I'll fix this next PR